### PR TITLE
ImageLoader: Force https for all wpcom images

### PR DESCRIPTION
**Pull request by maxme**

### CODE REVIEW

File: fluxc/src/main/java/org/wordpress/android/fluxc/tools/FluxCImageLoader.java [hunk 1]
```diff
@@ -42,9 +43,13 @@ public FluxCImageLoader(RequestQueue queue, ImageCache imageCache, AccessToken a
     }
 
     @Override
-    protected Request<Bitmap> makeImageRequest(final String requestUrl, int maxWidth, int maxHeight,
+    protected Request<Bitmap> makeImageRequest(String requestUrl, int maxWidth, int maxHeight,
                                                ScaleType scaleType, final String cacheKey) {
-        return new ImageRequest(requestUrl, new Response.Listener<Bitmap>() {
+        if (WPUrlUtils.isWordPressCom(requestUrl) && UrlUtils.isHttps(requestUrl)) {
```
Comment (aforcier) [inline fluxc/src/main/java/org/wordpress/android/fluxc/tools/FluxCImageLoader.java]: Shouldn't this be `!UrlUtils.isHttps(requestUrl)`?

File: fluxc/src/main/java/org/wordpress/android/fluxc/tools/FluxCImageLoader.java [hunk 2]
```diff
@@ -59,11 +64,11 @@ public void onErrorResponse(VolleyError error) {
             public Map<String, String> getHeaders() throws AuthFailureError {
                 HashMap<String, String> headers = new HashMap<String, String>();
```
Comment (aforcier) [inline fluxc/src/main/java/org/wordpress/android/fluxc/tools/FluxCImageLoader.java]: Not related to this diff, but the second `String, String` argument can be removed and make the linter green.

Comment (maxme) [inline fluxc/src/main/java/org/wordpress/android/fluxc/tools/FluxCImageLoader.java]: Updated in  	336e284

**Comment by aforcier** [comment]
:shipit:

**Pull request merged**